pkgs/by-name/so/sonarr/package.nix

```diff
@@ -69,8 +100,8 @@ buildDotnetModule {
 
   runtimeDeps = [ sqlite ];
 
-  dotnet-sdk = dotnetCorePackages.sdk_6_0-bin;
-  dotnet-runtime = dotnetCorePackages.aspnetcore_6_0-bin;
```

Comment:
Hm, can we use source-built sdk/runtimes for Sonarr (i.e. drop `-bin` suffix, see also #363008)?

Reply:
Will give it a shot

Reply:
Tested on aarch64-darwin, aarch64-linux and x86_64-linux, no apparent issues.